include/unifex/linux/io_uring_context.hpp

```diff
@@ -460,8 +461,47 @@ class io_uring_context::read_sender {
       }
     }
 
+    void request_stop() noexcept {
+      if (refCount_.fetch_add(1, std::memory_order_relaxed) == 0) {
```

Comment:
I wonder if it'd be cheaper to use a CAS here because the CPU might not need to write on failure.
```suggestion
      if (char expected = 1; refCount_.compare_exchange_strong(expected, 2, std::memory_order_relaxed)) {
```